tsk/base/tsk_base.h

```diff
@@ -103,6 +103,7 @@ extern "C" {
 #ifdef TSK_MULTITHREAD_LIB
 #ifdef TSK_WIN32
     void *tsk_error_win32_get_per_thread_(unsigned struct_size);
+    void tsk_error_win32_thread_cleanup();
```

Comment:
@bcarrier Is this adding the cleanup method to the public api?

Reply:
FYI, the comment on tsk_error_win32_thread_cleanup said "Threads must call this on exit to avoid a leak."
However I don't see any code calling it. VLD was reporting this leak.